jwt/utils.py

```diff
@@ -51,3 +61,51 @@ def uint_b64decode(uint_b64: str) -> int:
         value <<= 8
         value += int(b)
     return value
+
+
+def get_time(value: Union[str, int, datetime]) -> Union[int, datetime, None]:
+    """
+    :param value: The value of time to convert
+                  If is str or int datetime or None will be returned
+                  If is datetime int will be returned
+    :return: None if the string is not valid
+             datetime if is valid str or int
+             int if value is datetime
+    """
+    returned = None
+    if isinstance(value, str):
+        is_rfc3339 = RE_RFC3339.match(value)
+        if is_rfc3339:
```

Comment:
Is there any use case or mention in the JWT spec for RFC 3339? If not, I think converting any representation to `datetime` is the caller's responsibility.

Reply:
Because the [rfc7519](https://tools.ietf.org/html/rfc7519) reference to NumericDate related it to [rfc3339](https://tools.ietf.org/html/rfc3339) i create that function in order to allows convertions from all "expected types" to int.